tests/src/test/scala/scalaz/ValidationTest.scala

```diff
@@ -77,12 +78,12 @@ object ValidationTest extends SpecLite {
     "return first success" in {
       val succ = Validation.success[String, Int](1).toValidationNel
       val fail = Validation.failure[String, Int]("2").toValidationNel
-    
```

Comment:
> Or we should contribute more ;-)

+1, I am planning on it! :)

Personally I don't have an on-save hook, but my emacs config highlights the trailing space in bright red, and I set up a binding (`C-c d`) to run `delete-trailing-whitespace`.

Sure, I'll start a thread soon and see what people think if nobody beats me to it.